README.md

```diff
@@ -294,7 +294,7 @@ block's partial config.
 <details>
 <summary>How to compose blocks?</summary>
 
-Got some projects with similar, yet not identical webpack configurations? Create a “preset”, a function that returns a `group` of blocks so you could reuse it in several projects:
+Got some projects with similar, yet not identical webpack configurations? Create a “preset”, a function that returns a `group` of blocks so you could reuse it in multiple projects:
```

Comment:
I don't want to be that guy, but somehow I feel obligated to play the grammar nazi here 🙈😅
This really is a use case for ["can" rather than "could"](https://learnenglish.britishcouncil.org/en/english-grammar/verbs/modal-verbs/can-or-could).